### PR TITLE
test: updateUniqueVisitors function

### PR DESCRIPTION
**Pull request by Jemiiah**
# Pull Request type
Endpoint testing for - '/unique_page_visit?page_id=${id}' route

Please add the labels corresponding to the type of changes your PR introduces:

- Testing
Resolves: #611

### CODE REVIEW

File: tests/services/apiService.test.js [hunk 1]
```diff
@@ -137,3 +137,37 @@ describe("getBoostById function", () => {
...
+
+  it("should handle fetch errors gracefully", async () => {
+    const mockResponse = "Failed to deserialize query string: missing field `page_id`";
+    fetch.mockResolvedValueOnce({
+      json: () => Promise.resolve(mockResponse),
+    });
+
+    const result = await updateUniqueVisitors("3");
+    expect(fetch).toHaveBeenCalledWith(
+      `${API_URL}/unique_page_visit?page_id=${"3"}`
```
Comment (ayushtom) [inline tests/services/apiService.test.js:169]: if you remove the page id field can keep it as - `${API_URL}/unique_page_visit` then this error should come

Comment (ayushtom) [reply tests/services/apiService.test.js:169]: the url which you call is a valid request

**Comment by ayushtom** [comment]
@Jemiiah  i will be assigning someone else to finish this PR today. If you can then please commit the changes here so i can review and merge or else we give it to someone who can close the issue

**Comment by Jemiiah** [comment]
Sir please give me 20mins

**Pull request closed without merging**